### ISSUE TITLE
Issue with classification

### ISSUE THREAD

**Issue by ritwik12** [ritwik12/Virtual-Assistant#63]
After #53, when trying to say "find places to eat". It executes email functionality which it shouldn't.

**Comment by ritwik12** [comment]
@GaelleMarais and @LilyaMel Please help in fixing this.

**Comment by GaelleMarais** [comment]
Hi, I'll have a look

**Comment by ritwik12** [comment]
@GaelleMarais Thanks :)

**Comment by LilyaMel** [comment]
@ritwik12  We check the code but we don't have the problem that you describe. I send you a screen, we don't know why it works for us but not for you. Please tell us if it still here
![image](https://user-images.githubusercontent.com/47354050/57018548-4bedaa00-6c24-11e9-9584-eec2f65ab685.png)

**Comment by ritwik12** [comment]
You have tried this on new PR code or latest upstream code?

**Comment by LilyaMel** [comment]
Both and it works for the two of them

**Comment by GaelleMarais** [comment]
I can not reproduce the bug either
